TypeCobol.Test/Parser/Programs/Cobol85/SetAddrsOf.rdz.cbl

```diff
@@ -0,0 +1,51 @@
+﻿       IDENTIFICATION DIVISION.
+       PROGRAM-ID. SetAdrrOf.
+
+       DATA DIVISION.
+
+       WORKING-STORAGE SECTION.
+       01 W-myDate TYPE DATE.
+       01 W-myDate2 TYPE DATE.
+       01 W-PicVar PIC X(10).
+
+       linkage SECTION.
+       01 myDate TYPE DATE.
+       01 myDate2 TYPE DATE.
+       01 PicVar PIC X(10).
+       01 MyBool TYPE BOOL.
+
+
+       PROCEDURE DIVISION.
+
```

Comment:
Cobol doesn't allow to set an alphanumeric var or a typed var to something else.

These 3 lines are KO.

But perhaps it should be the subject of another issue.